src/arbeidssøkerskjema/Forside.less

```diff
@@ -0,0 +1,4 @@
+.nav-veileder {
+  justify-content: flex-end !important;
```

Comment:
Finnes det andre måter å løse spesifisitet på enn `!important`? 🤔

Reply:
Jeg wrappa Veileder-komponenten for å unngå det: 9f12930 :)